lib/storage/src/content_manager/collection_meta_ops.rs

```diff
@@ -298,6 +298,12 @@ pub enum ReshardingOperation {
         shard_id: ShardId,
         shard_key: Option<ShardKey>,
     },
+
+    Abort {
+        peer_id: PeerId,
```

Comment:
Do we need this params only to be sure we are not cancelling something outdated?

Reply:
Yes.